app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+  def create
+    @park = Park.find(params[:park_id])
+    @review = Review.new(review_params)
```

Comment:
associate the review to the park by doing this.

```ruby
@review = @park.reviews.new(review_params)
```
Following that you can also associate the signed in user here
```ruby
@review.user = current_user
```